index.js

```diff
@@ -179,6 +205,13 @@ module.exports = {
                 message: '`with` is disallowed in strict mode because it makes code impossible to predict and optimize.',
             }],
 
+        /*
+            Disallow returning values from setters.
+            This rule does not need to be explicilty set after enabled in airbnb:
+            https://github.com/airbnb/javascript/blob/5641278fa1df75ba3b6af2959a553bd682c0ad42/packages/eslint-config-airbnb-base/rules/errors.js#L121
+        */
+        'no-setter-return': 'error',
```

Comment:
This can be disabled for TypeScript, because it is covered by compiler error [2408](https://github.com/microsoft/TypeScript/blob/dc4cd52a9a09bd4eee1fbebde98b41f2c503b335/src/compiler/diagnosticMessages.json#L1242).

Reply:
This seems to be handled internally somehow. With the rule enabled in the index.js settings and doing the computed ruleset on a ts file I can see the rule resolves to being enabled. However when eslint runs on a ts file and in the editor it ignores the eslint rule (the eslint rule error for no-setter-return does not error but other eslint rules do):

![image](https://user-images.githubusercontent.com/1588923/118039546-13662800-b336-11eb-81bf-53a40153d8a8.png)

So I think typescript-eslint is taking care of that for us in some way.

Reply:
Interesting, maybe another rule that checks the configuration? No biggie, if it works it works.